app/src/test/java/org/simple/clinic/newentry/PatientEntryUpdateTest.kt

```diff
@@ -190,4 +190,30 @@ class PatientEntryUpdateTest {
         )
   }
 
+  @Test
+  fun `when the user enters date of birth in future, then show error`() {
+    //given
+    val errors: List<PatientEntryValidationError> = listOf(PatientEntryValidationError.DATE_OF_BIRTH_IN_FUTURE)
+    val givenModel = defaultModel
+        .fullNameChanged("Name")
+        .dateOfBirthChanged("02/02/4000")
```

Comment:
Since the validation is done using the `ageValidator`, which takes in a `UserClock`, we can set the date in the clock to a particular `LocalDate` (the default is the Unix EPOCH) and then set the date in the model to one which is after that to validate this.

I feel that this exposes the intent in this test more than just setting a date at some distant future.

We can pair on this if you're unclear about what to do.

Reply:
resolved